_layouts/mainhomepage.html

```diff
@@ -30,6 +30,9 @@
       }
     }
   </script>
+
+  <!-- Include the lang-links.html for canonical and alternate links -->
+  {% include lang-links.html %}
```

Comment:
Why just on the main homepage?

Reply:
Because I probably mistakenly assumed that's where it should go. As everything seems to work, I assume it's ok. If it belongs somewhere else, just tell me.

Reply:
I'd believe that the content in headtags.html (which gets included in every site) would suffice? Most likely, we'd have duplication here - but I'm not sure.

Reply:
@ann0see If I replace the canonical/alternate code in `headtags.html `with the content of `lang-links.html` (and remove the latter), and also remove the include from `mainhomepage.html`, everything works. I guess this is preferable as it's much tidier. If that sounds good to you I'll update this PR with it.

Reply:
Yes. And check if the content looks sensible in all pages.

Reply:
@ann0see done. While testing I found that switching back to EN after choosing another language wasn't working (in non-JS mode)... that's fixed. Everything looks good clicking through all the pages in all languages. The link at the bottom has been changed to the current latest artifact.